web-locks/resources/helpers.js

```diff
@@ -54,6 +54,7 @@
         resolve(event.data);
       };
       worker.addEventListener('message', listener);
+      worker.addEventListener('error', (e) => reject(e && e.message));
```

Comment:
nit: `(e)` -> `e` to match the style in the rest of the test

Reply:
Sure thing!